### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by jeyip**
screen shot 2017-08-30 at 12 31 01 am
screen shot 2017-08-30 at 12 31 09 am

### CODE REVIEW

File: web/client/src/views/Signup.js [hunk 1]
```diff
@@ -107,51 +90,89 @@ class Auth extends React.Component {
...
-              <a href="">New User</a>
-            </Form.Item>
           </Form>
         </div>
       </div>
     );
   }
 }
 
+Auth.defaultProps = {
```
Comment (mboperator) [inline web/client/src/views/Signup.js]: What's the reason for both using style variables, as well as CSS?

Comment (jeyip) [reply web/client/src/views/Signup.js]: Pseudo selectors, or properties that react inline styling won't assign like linear gradient

Comment (jeyip) [reply web/client/src/views/Signup.js]: I'm interested in sticking with styled components for speed up until we get a clearer picture of what can be shared.

Comment (jeyip) [reply web/client/src/views/Signup.js]: Then definitely scss / CSS modules

Comment (mboperator) [reply web/client/src/views/Signup.js]: Sweet! Let's try to do `styled-components` then

**Pull request merged**